course_catalog/etl/youtube_test.py

```diff
@@ -100,36 +108,21 @@ def mock_channel_data():
 
 
 @pytest.fixture
-def mocked_channel_response(mocked_responses, mock_channel_data, settings):
+@pytest.mark.usefixtures("video_settings")
```

Comment:
You don't need this if you also have it in the parameters, the use case for `usefixtures` is where you want to use the fixture but don't reference it in your test so it lets you avoid an unused param.